### PR TITLE
Techdebt

### PR DESCRIPTION
**Pull request by anissa-agahchen**
# Overview
* remove default logs
* fix tests
* fix code smells

### CODE REVIEW

File: api/src/paths/public/project/{projectId}/view.ts [hunk 1]
```diff
@@ -300,7 +300,7 @@ export function getPublicProjectForView(): RequestHandler {
       const projectService = new ProjectService(connection);
 
       const result = await projectService.getPublicProjectById(Number(req.params.projectId));
-
+      console.log('result is: ', result);
```
Comment (NickPhura) [inline api/src/paths/public/project/{projectId}/view.ts]: 👀

**Review by NickPhura — APPROVED** [review]

**Comment by NickPhura** [comment]
Just noticing 1 issue with the public view:
image

vs the admin view:
image

Activities and Project Type aren't rendering correctly on the public page.

Looks like the response from the API is returning the wrong values for those 2 fields:
image

**Review by NickPhura — CHANGES_REQUESTED** [review]
See my other comment (forgot to write it here originally)

**Comment by NickPhura** [comment]
> Just noticing 1 issue with the public view: image
> 
> vs the admin view: image
> 
> Activities and Project Type aren't rendering correctly on the public page.
> 
> Looks like the response from the API is returning the wrong values for those 2 fields: image

Pushed up a commit to address this

**Review by NickPhura — DISMISSED** [review]

**Comment by anissa-agahchen** [comment]
@NickPhura, thank you for the fix.

**Review by KjartanE — APPROVED** [review]
Looks good!

**Pull request merged**